drivers/wifi/winc1500/wifi_winc1500_nm_bsp.c

```diff
@@ -77,12 +77,14 @@ void nm_bsp_register_isr(void (*isr_fun)(void))
 void nm_bsp_interrupt_ctrl(u8_t enable)
 {
 	if (enable) {
-		gpio_pin_enable_callback(
+		gpio_pin_interrupt_configure(
```

Comment:
OK, but one function call with `enable ? GPIO_INT_ENABLE : GPIO_INT_DISABLE` as the value of the fourth parameter might be easier to maintain.

Has this been tested?  Normally we'd want something like `GPIO_INT_EDGE_TO_ACTIVE` to tell the driver what sort of interrupt to expect.  `GPIO_INT_ENABLE` may not select a valid configuration, since both LOW and HIGH are independent bits.

In fact, I think this would cause an assertion failure against this check:
```
        __ASSERT(((flags & GPIO_INT_ENABLE) == 0) ||
                 ((flags & (GPIO_INT_LOW_0 | GPIO_INT_HIGH_1)) != 0),
                 "At least one of GPIO_INT_LOW_0, GPIO_INT_HIGH_1 has to be "
                 "enabled.");
```
so it does have to be changed.

Reply:
> Has this been tested? 

No. I don't have the setup anymore and no time to put on that anyway.